src/middleware/rateLimiter.ts

```diff
@@ -0,0 +1,10 @@
+import rateLimit from 'express-rate-limit';
+import { Express } from '../types';
+
+export const rateLimiter: Express.MiddleWare = rateLimit({
```

Comment:
Is there a possible way we can allow specific IP addresses to not get stopped by the rate limiter? For instance, [Nkọwa okwu](https://nkowaokwu.com) uses the Igbo API and shouldn't be rate limited the same where an individual user should be rate limited

Reply:
Yes. I think this is posssible. We can have the IP addresses in a whitelist file. In the middleware,  we read the file to get these addresses then call the next function if the incoming request IP is same as any of the whitelisted IP address.

Does this make sense?

Reply:
yeah, that makes a lot of sense - do you know of any guides/tutorials/articles that walk through this process specifically for rate limiting node js / express apps?

i believe you can get a shortlist of valid GCP server IP addresses that we can whitelist since we are deploying this project on GCP